src/openloco/window.h

```diff
@@ -320,51 +323,58 @@ namespace openloco::ui
             uint8_t pad_all[0x88E];
```

Comment:
I think we should just get rid of this line and the encapsulating union entirely.